### DISCUSSION TITLE
Fetch previously published content

### DISCUSSION THREAD

**Discussion by krazyjakee** [directus/directus#4603] [Archive]
I would like for my users to be able to create drafts for content without changing the status of the content to draft and removing it from public view.

For example, a user wants to change the title, so they create a draft with that change. The website still shows the previous status of the content even though it now has a draft.

How would I do this?

**Comment by benhaynes** [comment]
I'll close this Discussion since it's a duplicate of: https://github.com/directus/directus/discussions/2975

We'll keep the conversation consolidated.
